tasks/sep-dev-0/006-holocene-sys-cfg/SignFromJson.s.sol

```diff
@@ -0,0 +1,97 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.15;
+
+import {SignFromJson as OriginalSignFromJson} from "script/SignFromJson.s.sol";
+import {Simulation} from "@base-contracts/script/universal/Simulation.sol";
+import {Types} from "@eth-optimism-bedrock/scripts/Types.sol";
+import {console2 as console} from "forge-std/console2.sol";
+import {stdJson} from "forge-std/StdJson.sol";
+import {LibString} from "solady/utils/LibString.sol";
+import {Vm, VmSafe} from "forge-std/Vm.sol";
+import "@eth-optimism-bedrock/src/dispute/lib/Types.sol";
+import {DisputeGameFactory} from "@eth-optimism-bedrock/src/dispute/DisputeGameFactory.sol";
+import {FaultDisputeGame} from "@eth-optimism-bedrock/src/dispute/FaultDisputeGame.sol";
+import {PermissionedDisputeGame} from "@eth-optimism-bedrock/src/dispute/PermissionedDisputeGame.sol";
+import {SystemConfig} from "@eth-optimism-bedrock/src/L1/SystemConfig.sol";
+import {Proxy} from "@eth-optimism-bedrock/src/universal/Proxy.sol";
+
+contract NestedSignFromJson is OriginalSignFromJson {
+    using LibString for string;
+
+    // Safe contract for this task.
+    address immutable proxyAdminOwnerSafe = vm.envAddress("OWNER_SAFE");
+    address immutable livenessGuard = 0x24424336F04440b1c28685a38303aC33C9D14a25;
```

Comment:
Let's read the guard from the storage slot instead of hardcoding the address. We almost ceratinly don't have this installed on devnet anyway, but I'm ok keeping it to minimize the diff between devnet and prod

You can see Adrian's use of the slot to fetch the guard here: https://github.com/ethereum-optimism/superchain-ops/blob/5bfc9ab82d4aacb854cacdb419b2862676fc7928/tasks/sep/fp-recovery/005-set-game-implementation/templates/NestedSignFromJson.s.sol#L33